portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -27,6 +27,6 @@
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     response.setContentType("text/html;");
-    response.getWriter().println("<h1>Utibeno</h1>");
+    response.getWriter().println("<h3>Welcome to my page</h3>");
```

Comment:
See my other comments, but I'm recommending that you remove the HTML tags from the servlet response.